src/vm/gdbjit.cpp

```diff
@@ -2231,10 +2229,7 @@ bool NotifyGdb::BuildSymbolTableSection(MemBuf& buf, PCODE addr, TADDR codeSize)
         sym[i].st_name = SymbolNames[i].m_off;
         sym[i].setBindingAndType(STB_GLOBAL, STT_FUNC);
         sym[i].st_other = 0;
-        sym[i].st_value = SymbolNames[i].m_value - addr;
-#ifdef _TARGET_ARM_
-        sym[i].st_value |= 1; // for THUMB code
-#endif    
+        sym[i].st_value = DataPointerToThumbCode<TADDR,PCODE>(SymbolNames[i].m_value - addr);
```

Comment:
You cannot do that for non-ARM32 builds, the function is defined for ARM32 only. Ideally, I'd suggest adding PINSTRToPCODE macro similar to PCODEToPINSTR, but working in the other direction and using it here.